src/views/dashboard/Home.tsx

```diff
@@ -85,12 +112,29 @@ export const HomeView = () => {
           </div>
         </div>
 
-        <h2 className="mx-auto mt-8 max-w-6xl px-4 text-lg font-medium leading-6 text-gray-900 sm:px-6 lg:px-8">
-          Recent syncs
-        </h2>
-        <h3 className="mx-auto mt-2 max-w-6xl px-4 text-lg font-medium leading-6 text-gray-400 sm:px-6 lg:px-8">
-          Coming soon
-        </h3>
+        <section className="mx-auto mt-10 max-w-6xl px-4 sm:px-6 lg:mt-12 lg:px-8">
+          <h2 className="text-lg font-medium leading-6 text-gray-900">
+            Recent syncs
+          </h2>
+          {syncs.syncs.length ? (
+            <div className="mt-4">
+              <SyncsList
+                syncs={syncs.syncs}
+                total={syncs.total}
+                page={syncs.page}
+                perPage={syncs.perPage}
+                onPageChange={(page) => setSyncs((prev) => ({ ...prev, page }))}
```

Comment:
`SyncsList` component is wrapped within memo, so `onPageChange` callback should have stable refference. (setSyncs method of setState have the same refference on each rerender so so the dependencies will be the same on each rerender)

Reply:
But onPageChange has stable reference here, and our dependencies changes will be reflected into component

Reply:
Yup, I did't see this condition `syncs.syncs.length` which cause that `SyncsList` render only once. But I think there's too much thinking here.
Read more here: https://attardi.org/why-we-memo-all-the-things/

Reply:
What you mean by "dependencies changes will be reflected into component"?